apps/app_queue.c

```diff
@@ -1783,6 +1783,11 @@ static int log_membername_as_agent;
 /*! \brief queues.conf [general] option */
 static int force_longest_waiting_caller;
 
+/*! \brief queues.conf [general] option */
+/*!< A switch to log Caller ID Name to param 4 when entering queue */
+/*!< Default: Do NOT log CID Name */
+static int log_caller_id_name = 0; 
```

Comment:
You don't need to assign `0` here. This can just be `static int log_caller_id_name;`